wezterm/src/main.rs

```diff
@@ -29,6 +29,10 @@ struct Opt {
     #[structopt(short = "n")]
     skip_config: bool,
 
+    /// Specify the configuration file to use, overrides WEZTERM_CONFIG_FILE
```

Comment:
I'd rather say `overrides the normal configuration file resolution` as that is a bit more precise